ui-ngx/src/app/core/http/domain.service.ts

```diff
@@ -33,8 +33,11 @@ export class DomainService {
   }
 
   public saveDomain(domain: Domain, oauth2ClientIds: Array<string>, config?: RequestConfig): Observable<Domain> {
```

Comment:
Consider making the oauth2ClientIds argument optional, as it might not always be required.